doc/mssql_examples_test.go

```diff
@@ -0,0 +1,61 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+	"log"
+	"time"
+
+	mssql "github.com/denisenkom/go-mssqldb"
```

Comment:
Don't need to explicitly set the alias. It is `mssql` by default.
```suggestion
	"github.com/denisenkom/go-mssqldb"
```

Reply:
I kept it since it seems to be like this in other examples.